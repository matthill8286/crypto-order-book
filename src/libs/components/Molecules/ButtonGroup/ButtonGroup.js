import React from 'react';
import { StyledButtonGroup } from "./ButtonGroup.styled";
export function ButtonGroup(props) {
  return React.createElement(StyledButtonGroup, null, props.children);
}
ButtonGroup.displayName = 'ButtonGroup';
//# sourceMappingURL=ButtonGroup.js.map