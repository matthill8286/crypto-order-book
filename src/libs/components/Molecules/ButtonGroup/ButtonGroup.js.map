{"version":3,"sources":["../../../../src/components/Molecules/ButtonGroup/ButtonGroup.tsx"],"names":["React","StyledButtonGroup","ButtonGroup","props","children","displayName"],"mappings":"AACA,OAAOA,KAAP,MAAkB,OAAlB;AACA,SAASC,iBAAT;AAEA,OAAO,SAASC,WAAT,CAAqBC,KAArB,EAAwE;AAC7E,SAAO,oBAAC,iBAAD,QAAoBA,KAAK,CAACC,QAA1B,CAAP;AACD;AAEDF,WAAW,CAACG,WAAZ,GAA0B,aAA1B","sourcesContent":["/* global JSX */\nimport React from 'react'\nimport { StyledButtonGroup } from './ButtonGroup.styled'\n\nexport function ButtonGroup(props: { children: React.ReactNode }): JSX.Element {\n  return <StyledButtonGroup>{props.children}</StyledButtonGroup>\n}\n\nButtonGroup.displayName = 'ButtonGroup'\n"],"file":"ButtonGroup.js"}