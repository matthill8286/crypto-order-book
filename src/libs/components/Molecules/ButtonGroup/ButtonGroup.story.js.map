{"version":3,"sources":["../../../../src/components/Molecules/ButtonGroup/ButtonGroup.story.tsx"],"names":["React","withState","storiesOf","ButtonGroup","Button","styled","stories","module","voidFn","StyledDivWithTopBorder","div","add","value","store","padding"],"mappings":"AAEA,OAAO,KAAKA,KAAZ,MAAuB,OAAvB;AACA,SAASC,SAAT,QAA0B,0BAA1B;AACA,SAASC,SAAT,QAA0B,kBAA1B;AACA,SAASC,WAAT;AACA,SAASC,MAAT;AACA,SAASC,MAAT;AAEA,IAAMC,OAAO,GAAGJ,SAAS,CAAC,qCAAD,EAAwCK,MAAxC,CAAzB;;AAEA,IAAMC,MAAM,GAAG,SAATA,MAAS,GAAM,CAAE,CAAvB;;AACA,IAAMC,sBAAsB,GAAGJ,MAAM,CAACK,GAAV;AAAA;AAAA;AAAA,+ZAA5B;AAgCAJ,OAAO,CAACK,GAAR,CACE,gBADF,EAEEV,SAAS,CAAC;AAAEW,EAAAA,KAAK,EAAE;AAAT,CAAD,EAAiB,UAAAC,KAAK,EAAI;AACjC,SACE;AAAK,IAAA,KAAK,EAAE;AAAEC,MAAAA,OAAO,EAAE;AAAX;AAAZ,KACE,oBAAC,WAAD,QACE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,UAAnB;AAA8B,IAAA,OAAO,EAAEN,MAAvC;AAA+C,IAAA,IAAI,EAAC,IAApD;AAAyD,IAAA,IAAI,EAAC;AAA9D,gBADF,EAIE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,UAAnB;AAA8B,IAAA,OAAO,EAAEA,MAAvC;AAA+C,IAAA,IAAI,EAAC,IAApD;AAAyD,IAAA,IAAI,EAAC;AAA9D,gBAJF,CADF,CADF;AAYD,CAbQ,CAFX;AAkBAF,OAAO,CAACK,GAAR,CACE,gBADF,EAEEV,SAAS,CAAC;AAAEW,EAAAA,KAAK,EAAE;AAAT,CAAD,EAAiB,UAAAC,KAAK,EAAI;AACjC,SACE;AAAK,IAAA,KAAK,EAAE;AAAEC,MAAAA,OAAO,EAAE;AAAX;AAAZ,KACE,oBAAC,WAAD,QACE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,UAAnB;AAA8B,IAAA,OAAO,EAAEN,MAAvC;AAA+C,IAAA,IAAI,EAAC,IAApD;AAAyD,IAAA,IAAI,EAAC;AAA9D,gBADF,EAIE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,UAAnB;AAA8B,IAAA,OAAO,EAAEA,MAAvC;AAA+C,IAAA,IAAI,EAAC,IAApD;AAAyD,IAAA,IAAI,EAAC;AAA9D,gBAJF,EAOE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,UAAnB;AAA8B,IAAA,OAAO,EAAEA,MAAvC;AAA+C,IAAA,IAAI,EAAC,IAApD;AAAyD,IAAA,IAAI,EAAC;AAA9D,gBAPF,CADF,EAYE,+BAZF,EAaE,oBAAC,WAAD,QACE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,WAAnB;AAA+B,IAAA,OAAO,EAAEA,MAAxC;AAAgD,IAAA,IAAI,EAAC,IAArD;AAA0D,IAAA,IAAI,EAAC;AAA/D,gBADF,EAIE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,WAAnB;AAA+B,IAAA,OAAO,EAAEA,MAAxC;AAAgD,IAAA,IAAI,EAAC,IAArD;AAA0D,IAAA,IAAI,EAAC;AAA/D,gBAJF,EAOE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,WAAnB;AAA+B,IAAA,OAAO,EAAEA,MAAxC;AAAgD,IAAA,IAAI,EAAC,IAArD;AAA0D,IAAA,IAAI,EAAC;AAA/D,gBAPF,CAbF,EAwBE,+BAxBF,EAyBE,oBAAC,WAAD,QACE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,WAAnB;AAA+B,IAAA,OAAO,EAAEA,MAAxC;AAAgD,IAAA,IAAI,EAAC,IAArD;AAA0D,IAAA,IAAI,EAAC;AAA/D,gBADF,EAIE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,WAAnB;AAA+B,IAAA,OAAO,EAAEA,MAAxC;AAAgD,IAAA,IAAI,EAAC,IAArD;AAA0D,IAAA,IAAI,EAAC;AAA/D,gBAJF,EAOE,oBAAC,MAAD;AAAQ,IAAA,UAAU,EAAC,WAAnB;AAA+B,IAAA,OAAO,EAAEA,MAAxC;AAAgD,IAAA,IAAI,EAAC,IAArD;AAA0D,IAAA,IAAI,EAAC;AAA/D,gBAPF,CAzBF,CADF;AAuCD,CAxCQ,CAFX","sourcesContent":["/* eslint-disable @typescript-eslint/ban-ts-comment */\n// @ts-nocheck => JSX property style does not exists in Typo component\nimport * as React from 'react'\nimport { withState } from '@dump247/storybook-state'\nimport { storiesOf } from '@storybook/react'\nimport { ButtonGroup } from './ButtonGroup'\nimport { Button } from '@/components/Atoms/Button'\nimport { styled } from '@/styles/styled'\n\nconst stories = storiesOf('Design System/Molecules/ButtonGroup', module)\n\nconst voidFn = () => {}\nconst StyledDivWithTopBorder = styled.div`\n  padding-top: 25px;\n  position: relative;\n  background-color: #fff;\n  border-top: 1px solid #000;\n\n  &:after,\n  &:before {\n    top: -1px;\n    left: 7px;\n    border: solid transparent;\n    content: '';\n    height: 0;\n    width: 0;\n    position: absolute;\n    pointer-events: none;\n  }\n\n  &:after {\n    border-color: rgba(136, 183, 213, 0);\n    border-top-color: #fff;\n    border-width: 8px;\n    margin-left: -8px;\n  }\n  &:before {\n    border-color: rgba(194, 225, 245, 0);\n    border-top-color: #000;\n    border-width: 9px;\n    margin-left: -9px;\n  }\n`\n\nstories.add(\n  'With 2 Buttons',\n  withState({ value: '1' }, store => {\n    return (\n      <div style={{ padding: '20px' }}>\n        <ButtonGroup>\n          <Button actionType=\"outlined\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 1\n          </Button>\n          <Button actionType=\"outlined\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 2\n          </Button>\n        </ButtonGroup>\n      </div>\n    )\n  })\n)\n\nstories.add(\n  'With 3 Buttons',\n  withState({ value: '1' }, store => {\n    return (\n      <div style={{ padding: '20px' }}>\n        <ButtonGroup>\n          <Button actionType=\"outlined\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 1\n          </Button>\n          <Button actionType=\"outlined\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 2\n          </Button>\n          <Button actionType=\"outlined\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 3\n          </Button>\n        </ButtonGroup>\n        <br />\n        <ButtonGroup>\n          <Button actionType=\"secondary\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 1\n          </Button>\n          <Button actionType=\"secondary\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 2\n          </Button>\n          <Button actionType=\"secondary\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 3\n          </Button>\n        </ButtonGroup>\n        <br />\n        <ButtonGroup>\n          <Button actionType=\"prominent\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 1\n          </Button>\n          <Button actionType=\"prominent\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 2\n          </Button>\n          <Button actionType=\"prominent\" onClick={voidFn} size=\"sm\" type=\"button\">\n            Button 3\n          </Button>\n        </ButtonGroup>\n      </div>\n    )\n  })\n)\n"],"file":"ButtonGroup.story.js"}