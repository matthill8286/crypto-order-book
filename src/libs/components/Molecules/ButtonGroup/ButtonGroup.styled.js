import { styled, css } from "../../../styles";

var color = function color(x) {
  return function (_ref) {
    var theme = _ref.theme;
    return theme.color[x];
  };
};

var spacing = function spacing(x) {
  return function (_ref2) {
    var theme = _ref2.theme;
    return theme.spacing.base[x];
  };
};

var innerShadowColor = function innerShadowColor(strColor) {
  return "inset 0px 0px 0px 1px ".concat(strColor);
};

var innerShadowFromTheme = function innerShadowFromTheme(x) {
  return function (_ref3) {
    var theme = _ref3.theme;
    return innerShadowColor(theme.color[x]);
  };
};

export var StyledButtonGroup = styled.div.withConfig({
  displayName: "ButtonGroupstyled__StyledButtonGroup",
  componentId: "sc-1qkc8ga-0"
})(["display:flex;width:100%;background-color:", ";button,[type='button']{border-radius:0;flex:1 1 auto;&:first-child{border-top-left-radius:4px;border-bottom-left-radius:4px;}&:last-child{border-top-right-radius:4px;border-bottom-right-radius:4px;}}"], color('white'));
var StyledButton = styled.button.attrs({
  type: 'button'
}).withConfig({
  displayName: "ButtonGroupstyled__StyledButton",
  componentId: "sc-1qkc8ga-1"
})(["cursor:pointer;box-sizing:border-box;min-width:50px;padding:", ";border-width:1px;border-style:solid;border-color:", ";border-radius:", ";background-color:", ";transition-duration:0.15s;transition-timing-function:ease-in-out;transition-property:border-color,box-shadow;outline:none;&:hover:enabled,&:active:enabled{box-shadow:", ";border-color:", ";}&:disabled{cursor:not-allowed;}"], spacing('xs'), color('grey2'), spacing('xxs'), color('white'), innerShadowFromTheme('primary'), color('primary'));
export var BorderedButton = styled(StyledButton).withConfig({
  displayName: "ButtonGroupstyled__BorderedButton",
  componentId: "sc-1qkc8ga-2"
})(function (_ref4) {
  var selected = _ref4.selected,
      theme = _ref4.theme;
  return css(["", ""], Boolean(selected) && css(["border-color:", ";box-shadow:", ";"], theme.color.primary, innerShadowColor(theme.color.primary)));
});
//# sourceMappingURL=ButtonGroup.styled.js.map