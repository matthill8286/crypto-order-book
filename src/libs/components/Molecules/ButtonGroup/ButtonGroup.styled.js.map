{"version":3,"sources":["../../../../src/components/Molecules/ButtonGroup/ButtonGroup.styled.tsx"],"names":["styled","css","color","x","theme","spacing","base","innerShadowColor","strColor","innerShadowFromTheme","StyledButtonGroup","div","StyledButton","button","attrs","type","BorderedButton","selected","Boolean","primary"],"mappings":"AAAA,SAASA,MAAT,EAAiBC,GAAjB;;AAKA,IAAMC,KAAK,GAAG,SAARA,KAAQ,CAACC,CAAD;AAAA,SAAoB;AAAA,QAAGC,KAAH,QAAGA,KAAH;AAAA,WAAeA,KAAK,CAACF,KAAN,CAAYC,CAAZ,CAAf;AAAA,GAApB;AAAA,CAAd;;AACA,IAAME,OAAO,GAAG,SAAVA,OAAU,CAACF,CAAD;AAAA,SAAmB;AAAA,QAAGC,KAAH,SAAGA,KAAH;AAAA,WAAeA,KAAK,CAACC,OAAN,CAAcC,IAAd,CAAmBH,CAAnB,CAAf;AAAA,GAAnB;AAAA,CAAhB;;AACA,IAAMI,gBAAgB,GAAG,SAAnBA,gBAAmB,CAACC,QAAD;AAAA,yCAA+CA,QAA/C;AAAA,CAAzB;;AACA,IAAMC,oBAAoB,GAAG,SAAvBA,oBAAuB,CAACN,CAAD;AAAA,SAAoB;AAAA,QAAGC,KAAH,SAAGA,KAAH;AAAA,WAAeG,gBAAgB,CAACH,KAAK,CAACF,KAAN,CAAYC,CAAZ,CAAD,CAA/B;AAAA,GAApB;AAAA,CAA7B;;AAEA,OAAO,IAAMO,iBAAiB,GAAGV,MAAM,CAACW,GAAV;AAAA;AAAA;AAAA,8PAGRT,KAAK,CAAC,OAAD,CAHG,CAAvB;AAoBP,IAAMU,YAAY,GAAGZ,MAAM,CAACa,MAAP,CAAcC,KAAd,CAAoB;AAAEC,EAAAA,IAAI,EAAE;AAAR,CAApB,CAAH;AAAA;AAAA;AAAA,sYAILV,OAAO,CAAC,IAAD,CAJF,EAOAH,KAAK,CAAC,OAAD,CAPL,EAQCG,OAAO,CAAC,KAAD,CARR,EASIH,KAAK,CAAC,OAAD,CATT,EAmBAO,oBAAoB,CAAC,SAAD,CAnBpB,EAoBEP,KAAK,CAAC,SAAD,CApBP,CAAlB;AA0BA,OAAO,IAAMc,cAAc,GAAGhB,MAAM,CAACY,YAAD,CAAT;AAAA;AAAA;AAAA,GACzB;AAAA,MAAGK,QAAH,SAAGA,QAAH;AAAA,MAAab,KAAb,SAAaA,KAAb;AAAA,SAAyBH,GAAzB,WACIiB,OAAO,CAACD,QAAD,CAAP,IACAhB,GADA,yCAEkBG,KAAK,CAACF,KAAN,CAAYiB,OAF9B,EAGgBZ,gBAAgB,CAACH,KAAK,CAACF,KAAN,CAAYiB,OAAb,CAHhC,CADJ;AAAA,CADyB,CAApB","sourcesContent":["import { styled, css } from '@/styles'\nimport { Color, Size } from '@/types'\n\nexport type StyledOptionProps = { selected?: boolean }\n\nconst color = (x: keyof Color) => ({ theme }) => theme.color[x]\nconst spacing = (x: keyof Size) => ({ theme }) => theme.spacing.base[x]\nconst innerShadowColor = (strColor: string) => `inset 0px 0px 0px 1px ${strColor}`\nconst innerShadowFromTheme = (x: keyof Color) => ({ theme }) => innerShadowColor(theme.color[x])\n\nexport const StyledButtonGroup = styled.div`\n  display: flex;\n  width: 100%;\n  background-color: ${color('white')};\n  button,\n  [type='button'] {\n    border-radius: 0;\n    flex: 1 1 auto;\n\n    &:first-child {\n      border-top-left-radius: 4px;\n      border-bottom-left-radius: 4px;\n    }\n    &:last-child {\n      border-top-right-radius: 4px;\n      border-bottom-right-radius: 4px;\n    }\n  }\n`\n\nconst StyledButton = styled.button.attrs({ type: 'button' })`\n  cursor: pointer;\n  box-sizing: border-box;\n  min-width: 50px;\n  padding: ${spacing('xs')};\n  border-width: 1px;\n  border-style: solid;\n  border-color: ${color('grey2')};\n  border-radius: ${spacing('xxs')};\n  background-color: ${color('white')};\n\n  transition-duration: 0.15s;\n  transition-timing-function: ease-in-out;\n  transition-property: border-color, box-shadow;\n\n  outline: none;\n\n  &:hover:enabled,\n  &:active:enabled {\n    box-shadow: ${innerShadowFromTheme('primary')};\n    border-color: ${color('primary')};\n  }\n  &:disabled {\n    cursor: not-allowed;\n  }\n`\nexport const BorderedButton = styled(StyledButton)<StyledOptionProps>(\n  ({ selected, theme }) => css`\n    ${Boolean(selected) &&\n      css`\n        border-color: ${theme.color.primary};\n        box-shadow: ${innerShadowColor(theme.color.primary)};\n      `}\n  `\n)\n"],"file":"ButtonGroup.styled.js"}