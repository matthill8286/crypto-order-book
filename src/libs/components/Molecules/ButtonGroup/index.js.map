{"version":3,"sources":["../../../../src/components/Molecules/ButtonGroup/index.ts"],"names":[],"mappings":"AAAA","sourcesContent":["export * from './ButtonGroup'\n"],"file":"index.js"}