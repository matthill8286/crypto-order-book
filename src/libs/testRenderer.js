import { mount } from 'enzyme';
import React from 'react';
import { MemoryRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import { saiyanTheme } from "./styles/sc-vars-saiyan";
export function renderWithTheme(tree) {
  var theme = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : saiyanTheme;
  return renderer.create(React.createElement(ThemeProvider, {
    theme: theme
  }, tree)).toJSON();
}
export function renderWithThemeAndRouter(tree) {
  var theme = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : saiyanTheme;
  return renderer.create(React.createElement(ThemeProvider, {
    theme: theme
  }, React.createElement(Router, null, tree))).toJSON();
}
export function renderWithRouter(tree) {
  return renderer.create(React.createElement(Router, null, tree)).toJSON();
}
export function mountWithTheme(tree) {
  return mount(React.createElement(ThemeProvider, {
    theme: saiyanTheme
  }, tree));
}
export function mountWithThemeAndRouter(tree) {
  return mount(React.createElement(ThemeProvider, {
    theme: saiyanTheme
  }, React.createElement(Router, null, tree)));
}
//# sourceMappingURL=testRenderer.js.map