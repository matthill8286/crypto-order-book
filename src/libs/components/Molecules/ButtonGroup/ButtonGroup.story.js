import * as React from 'react';
import { withState } from '@dump247/storybook-state';
import { storiesOf } from '@storybook/react';
import { ButtonGroup } from "./ButtonGroup";
import { Button } from "../../Atoms/Button";
import { styled } from "../../../styles/styled";
var stories = storiesOf('Design System/Molecules/ButtonGroup', module);

var voidFn = function voidFn() {};

var StyledDivWithTopBorder = styled.div.withConfig({
  displayName: "ButtonGroupstory__StyledDivWithTopBorder",
  componentId: "y9vh3n-0"
})(["padding-top:25px;position:relative;background-color:#fff;border-top:1px solid #000;&:after,&:before{top:-1px;left:7px;border:solid transparent;content:'';height:0;width:0;position:absolute;pointer-events:none;}&:after{border-color:rgba(136,183,213,0);border-top-color:#fff;border-width:8px;margin-left:-8px;}&:before{border-color:rgba(194,225,245,0);border-top-color:#000;border-width:9px;margin-left:-9px;}"]);
stories.add('With 2 Buttons', withState({
  value: '1'
}, function (store) {
  return React.createElement("div", {
    style: {
      padding: '20px'
    }
  }, React.createElement(ButtonGroup, null, React.createElement(Button, {
    actionType: "outlined",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 1"), React.createElement(Button, {
    actionType: "outlined",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 2")));
}));
stories.add('With 3 Buttons', withState({
  value: '1'
}, function (store) {
  return React.createElement("div", {
    style: {
      padding: '20px'
    }
  }, React.createElement(ButtonGroup, null, React.createElement(Button, {
    actionType: "outlined",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 1"), React.createElement(Button, {
    actionType: "outlined",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 2"), React.createElement(Button, {
    actionType: "outlined",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 3")), React.createElement("br", null), React.createElement(ButtonGroup, null, React.createElement(Button, {
    actionType: "secondary",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 1"), React.createElement(Button, {
    actionType: "secondary",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 2"), React.createElement(Button, {
    actionType: "secondary",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 3")), React.createElement("br", null), React.createElement(ButtonGroup, null, React.createElement(Button, {
    actionType: "prominent",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 1"), React.createElement(Button, {
    actionType: "prominent",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 2"), React.createElement(Button, {
    actionType: "prominent",
    onClick: voidFn,
    size: "sm",
    type: "button"
  }, "Button 3")));
}));
//# sourceMappingURL=ButtonGroup.story.js.map