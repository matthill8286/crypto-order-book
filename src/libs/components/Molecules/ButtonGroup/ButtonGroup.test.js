import React from 'react';
import { renderWithTheme } from "../../../testRenderer";
import { ButtonGroup } from "./ButtonGroup";
describe('ButtonGroup', function () {
  it('renders correctly', function () {
    var component = renderWithTheme(React.createElement(ButtonGroup, null, "Hello"));
    expect(component).toMatchSnapshot();
  });
});
//# sourceMappingURL=ButtonGroup.test.js.map