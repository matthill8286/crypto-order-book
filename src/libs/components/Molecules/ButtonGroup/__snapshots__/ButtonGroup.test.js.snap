// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup renders correctly 1`] = `
<div
  className="ButtonGroupstyled__StyledButtonGroup-sc-1qkc8ga-0 kapciZ"
>
  Hello
</div>
`;
